flumine/markets/blotter.py

```diff
@@ -255,9 +255,11 @@ def __setitem__(self, customer_order_ref: str, order) -> None:
         self._orders[customer_order_ref] = order
         self._bet_id_lookup[order.bet_id] = order
         self._live_orders.append(order)
+        self._live_orders = _sort_orders(self._live_orders)
         strategy = order.trade.strategy
         self._trades[order.trade.id].append(order)
         self._strategy_orders[strategy].append(order)
+        self._strategy_orders[strategy] = _sort_orders(self._strategy_orders[strategy])
         self._strategy_selection_orders[
             (strategy, order.selection_id, order.handicap)
         ].append(order)
```

Comment:
client_orders needs to be sorted as well, seeing issues with multi clients when testing